test/Directory.build.props

```diff
@@ -0,0 +1,42 @@
+<Project>
+  <PropertyGroup>
+    <DisableImplicitFrameworkReferences Condition=" '$(TargetFramework)' == 'net452' ">true</DisableImplicitFrameworkReferences>
+  </PropertyGroup>
+
+  <PropertyGroup Condition=" '$(TargetFramework)' == 'netcoreapp1.1' ">

```

Comment:
¯\_(ツ)_/¯

I reckoned that we define shared stuff in this file and it will be used by projects based on their supported target frameworks. Ideally all of our test projects can be changed to support both `net451` and `netstandard`